geniza/corpus/tests/test_corpus_views.py

```diff
@@ -255,6 +315,76 @@ def test_get_context_data(self, rf):
             == docsearch_view.queryset.get_highlighting.return_value
         )
 
+    def test_scholarship_sort(self, empty_solr, source):
+        """integration test for sorting by scholarship asc and desc"""
+
+        doc_no_record = Document.objects.create()
+        doc_one_record = Document.objects.create(description="My record count is one")
+        Footnote.objects.create(
+            content_object=doc_one_record,
+            source=source,
+            doc_relation=Footnote.EDITION,
+        )
+        assert doc_one_record.index_data()["scholarship_count_i"] == 1
```

Comment:
this should be tested elsewhere

Reply:
You're right—already tested in `test_corpus_models`!